application/aws/aws.go

```diff
@@ -146,3 +143,28 @@ func GetFileURL(key string) (ObjectUrl, error) {
 
 	return getObjectURL(config, svc, key)
 }
+
+// CreateS3Bucket creates an S3 bucket with a name defined by an environment variable. If the bucket already
+// exists, it will not return an error.
+func CreateS3Bucket() error {
+	config := GetS3ConfigFromEnv()
+
+	svc, err := CreateS3Service(config)
+	if err != nil {
+		return err
+	}
+
+	if _, err := svc.CreateBucket(&s3.CreateBucketInput{
```

Comment:
Maybe have the CreateBucketInput done in the line above, to improve readability?

Reply:
The bucket should be created by Terraform along with the user and appropriate policy to restrict user access and then fed into the environment

Reply:
Good point. Nowhere in the app is this called. I'll try moving it back to test code, but I've had trouble in the past accessing test code from another package. I guess I could just create two copies of this code.